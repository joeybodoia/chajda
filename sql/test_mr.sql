CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."');
