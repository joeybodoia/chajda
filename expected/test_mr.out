CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                               spacy_lemmatize                                               
-------------------------------------------------------------------------------------------------------------
   माझे:3 उदाहरण:4 चाचणी:5 वाक्य:6 चाचणी:10 प्रकरणे:11 व्युत्पन्न:12 करण्यासाठी:13 google:14 भाषांतर:15 मध्ये:16 टाकत:17
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                               spacy_lemmatize                               
-----------------------------------------------------------------------------
   माझे उदाहरण चाचणी वाक्य चाचणी प्रकरणे व्युत्पन्न करण्यासाठी google भाषांतर मध्ये टाकत
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                           spacy_lemmatize                                                           
-------------------------------------------------------------------------------------------------------------------------------------
   हे:2 माझे:3 उदाहरण:4 चाचणी:5 वाक्य:6 आहे:7 जे:8 मी:9 चाचणी:10 प्रकरणे:11 व्युत्पन्न:12 करण्यासाठी:13 google:14 भाषांतर:15 मध्ये:16 टाकत:17 आहे:18
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                     spacy_lemmatize                                      
------------------------------------------------------------------------------------------
   हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी google भाषांतर मध्ये टाकत आहे
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                      spacy_lemmatize                                                      
---------------------------------------------------------------------------------------------------------------------------
   ":2 माझे:4 उदाहरण:5 चाचणी:6 वाक्य:7 चाचणी:11 प्रकरणे:12 व्युत्पन्न:13 करण्यासाठी:14 google:15 भाषांतर:16 मध्ये:17 टाकत:18 .:20 ":21
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                  spacy_lemmatize                                  
-----------------------------------------------------------------------------------
   " माझे उदाहरण चाचणी वाक्य चाचणी प्रकरणे व्युत्पन्न करण्यासाठी google भाषांतर मध्ये टाकत . "
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                  spacy_lemmatize                                                                   
----------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 हे:3 माझे:4 उदाहरण:5 चाचणी:6 वाक्य:7 आहे:8 जे:9 मी:10 चाचणी:11 प्रकरणे:12 व्युत्पन्न:13 करण्यासाठी:14 google:15 भाषांतर:16 मध्ये:17 टाकत:18 आहे:19 .:20 ":21
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                        spacy_lemmatize                                         
------------------------------------------------------------------------------------------------
   " हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी google भाषांतर मध्ये टाकत आहे . "
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                               spacy_lemmatize                                               
-------------------------------------------------------------------------------------------------------------
   माझे:3 उदाहरण:4 चाचणी:5 वाक्य:6 चाचणी:10 प्रकरणे:11 व्युत्पन्न:12 करण्यासाठी:13 Google:14 भाषांतर:15 मध्ये:16 टाकत:17
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                               spacy_lemmatize                               
-----------------------------------------------------------------------------
   माझे उदाहरण चाचणी वाक्य चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                           spacy_lemmatize                                                           
-------------------------------------------------------------------------------------------------------------------------------------
   हे:2 माझे:3 उदाहरण:4 चाचणी:5 वाक्य:6 आहे:7 जे:8 मी:9 चाचणी:10 प्रकरणे:11 व्युत्पन्न:12 करण्यासाठी:13 Google:14 भाषांतर:15 मध्ये:16 टाकत:17 आहे:18
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                     spacy_lemmatize                                      
------------------------------------------------------------------------------------------
   हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                      spacy_lemmatize                                                      
---------------------------------------------------------------------------------------------------------------------------
   ":2 माझे:4 उदाहरण:5 चाचणी:6 वाक्य:7 चाचणी:11 प्रकरणे:12 व्युत्पन्न:13 करण्यासाठी:14 Google:15 भाषांतर:16 मध्ये:17 टाकत:18 .:20 ":21
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                  spacy_lemmatize                                  
-----------------------------------------------------------------------------------
   " माझे उदाहरण चाचणी वाक्य चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत . "
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                  spacy_lemmatize                                                                   
----------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 हे:3 माझे:4 उदाहरण:5 चाचणी:6 वाक्य:7 आहे:8 जे:9 मी:10 चाचणी:11 प्रकरणे:12 व्युत्पन्न:13 करण्यासाठी:14 Google:15 भाषांतर:16 मध्ये:17 टाकत:18 आहे:19 .:20 ":21
(1 row)

SELECT spacy_lemmatize('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                        spacy_lemmatize                                         
------------------------------------------------------------------------------------------------
   " हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे . "
(1 row)

SELECT spacy_tsvector('mr',' "हे माझे उदाहरण चाचणी वाक्य आहे जे मी चाचणी प्रकरणे व्युत्पन्न करण्यासाठी Google भाषांतर मध्ये टाकत आहे."');
                                                      spacy_tsvector                                                       
---------------------------------------------------------------------------------------------------------------------------
 'google':14 'उदाहरण':4 'करण्यासाठी':13 'चाचणी':5,10 'टाकत':17 'प्रकरणे':11 'भाषांतर':15 'मध्ये':16 'माझे':3 'वाक्य':6 'व्युत्पन्न':12
(1 row)

